pkg/installmanager/installmanager_test.go

```diff
@@ -723,6 +724,42 @@ func Test_pasteInPullSecret(t *testing.T) {
 	}
 }
 
+func Test_nutanix_pasteInProviderCredentials(t *testing.T) {
```

Comment:
Nice test. If you're feeling ambitious, you could extend it to cover
- the error paths
- the path where the creds are already correct in the input IC.

Reply:
Done